features/deploy.feature

```diff
@@ -5,24 +5,16 @@ Feature: The deploy command
     Unfortunately, Aruba cannot run commands truly interactively. We need to
     answer prompts blindly, and check the output afterwards.
 
-    When I run `geordi deploy` interactively
-      # Answer three prompts
-      And I type "staging"
-      And I type "master"
-      And I type ""
-      # Confirm deployment
-      And I type "yes"
+    When I run `geordi deploy` in interactive mode
+    Then I should see a prompt "Deployment stage: [staging]"
```

Comment:
These changes would come at a high cost. Unfortunately, the command does not seem to write to stdout as long as it is still running. Therefore, to make this work, we'd have to stop the command, and start it again from beginning to make this work. See the step definitions below. This Feature alone in this state would take a minute to run, so I think this should be rolled back, but I wanted to show it and see what you think

Reply:
Thanks for the proof of concept, but I agree this is too high an overhead. Please revert.